WTWSMS/common/nicknames/VIET_immersion_nicknames.txt

```diff
@@ -774,7 +790,6 @@ nick_the_philosopher = {
 			trait = imbecile
 			trait = inbred
 			trait = incapable
-			religion = catholic
 			religion_group = pagan_group
```

Comment:
Again, this is about having unique nicknames for distinct flavour. Catholicism is not the focus of the mod, but it doesn't either focus on Philosophy.